tmva/sofie/inc/TMVA/ROperator_BatchNormalization.hxx

```diff
@@ -107,12 +107,12 @@ public:
 
 		fShapeX = model.GetTensorShape(fNX);
       
-      if (fShapeX.size() <  2 || fShapeX.size() > 4) {
-         throw
-				std::runtime_error("TMVA SOFIE BatchNormalization Op input tensor " + fNX + " fnx has wrong shape : " + ConvertShapeToString(fShapeX));
-      }
+		if (fShapeX.size() <  2 || fShapeX.size() > 4) {
```

Comment:
The indentation here should be only 3 spaces and no tabs. 
I think also above is not correct. Could you add this fix too, please ?

Reply:
Yes Done.